### PR TITLE
[Android] Fix issue calculating the threshold using SwipeItemViews 

### LINKED ISSUE #13547: [Android] SwipeItemView size issues
### Description

In version 7.0.58 you have pushed some changes to fix SwipeItemView size problems (https://github.com/dotnet/maui/pull/10522).

But there is still one problem with this and it is when you programmatically triggers to open the SwipeView.

Opened with swipe gesture:
image

Opened programmatically:
image

It leaves some blank space between the content and the Swipe items.

### Steps to Reproduce

1. Create a new project
2. Add a SwipeView to the content of the page
3. Add a Button to trigger the swipe programmatically
4. You will see the difference between both behaviours

Expected outcome: both with a gesture or programmatically, the swipe should behave the same way and no blank space should be displayed
Actual outcome: When opening it programmatically, there is a blank space between SwipeView content and SwipeView items

### Link to public reproduction project repository

https://github.com/nicop85/SwipeViewSizeProblems

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11 and up

### Did you find any workaround?

No

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Fix issue calculating the threshold using SwipeItemViews on Android.

fix-open-prog-swipeview

### Issues Fixed

Fixes #13547

### CODE REVIEW

File: src/Core/src/Platform/Android/MauiSwipeView.cs [hunk 1]
```diff
@@ -1344,11 +1344,12 @@ void ProgrammaticallyOpenSwipeItem(OpenSwipeItem openSwipeItem, bool animated)
 
 			if (swipeItems == null || swipeItems.Count == 0)
 				return;
+			
+			UpdateSwipeItems();
```
Comment (jsuarezruiz) [inline src/Core/src/Platform/Android/MauiSwipeView.cs:1348]: With the changes, we measure the SwipeItemViews before getting the threshold. What happened was that the size obtained from each SwipeItemView was zero and the default thjreshold value was used.

**Review by PureWeen — CHANGES_REQUESTED** [review]
Can we add a test? 

When opening the swipeview programmatically we can see if the size matches what we expect?

**Review by mandel-macaque — APPROVED** [review]
Approving as long as @PureWeen are considered.

**Comment by jsuarezruiz** [comment]
> Can we add a test?
> 
> When opening the swipeview programmatically we can see if the size matches what we expect?

Added different device tests.

**Review by rmarinho — APPROVED** [review]

**Review by PureWeen — APPROVED** [review]

**Pull request merged**